app-alternatives/vi/metadata.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "https://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<maintainer type="project">
+		<email>base-system@gentoo.org</email>
+		<name>Gentoo Base System</name>
+	</maintainer>
+	<maintainer type="person">
+		<email>mgorny@gentoo.org</email>
+		<name>Michał Górny</name>
+	</maintainer>
+	<use>
+		<flag name="busybox">Symlink to <pkg>sys-apps/busybox</pkg></flag>
+		<flag name="elvis">Symlink to <pkg>app-editors/elvis</pkg></flag>
```

Comment:
Elvis will be dropped https://archives.gentoo.org/gentoo-dev/message/98903f6fcba36049437f764709d5bd29

Reply:
I know. However, I'm waiting with the update in case one of usual necro… err, developers, decided that we must keep it.